spec/spec_helper.rb

```diff
@@ -3,7 +3,7 @@
 require 'openstates'
 require 'pry'
 require 'pry-remote'
-require 'pry-debugger'
+require 'pry-byebug'
```

Comment:
I like single quotes also :D

I tried to keep the styling the way it was before.